### PR TITLE
feat: langchain integration for greptimeai

### PR DESCRIPTION
**Pull request by yuanbohan**
## What's Changed in this PR

langchain integration for greptimeai

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/en/v0.4/greptimecloud/greptimeai/langchain.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## installation
```
Comment (sunng87) [inline docs/en/v0.4/greptimecloud/greptimeai/langchain.md:1]: Capitalize

File: docs/en/v0.4/greptimecloud/greptimeai/langchain.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Installation
+
+```python
+pip install greptimeai
+```
+
+## Setup
```
Comment (sunng87) [inline docs/en/v0.4/greptimecloud/greptimeai/langchain.md:7]: Add textual description for each step

File: docs/en/v0.4/greptimecloud/greptimeai/langchain.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+export GREPTIMEAI_DATABASE='<dbname>'
+export GREPTIMEAI_TOKEN='<username>:<password>'
+```
+
+## How-To
+
+```python
+from greptimeai.langchain.callback import GreptimeCallbackHandler
+
+greptime_callback = GreptimeCallbackHandler()
```
Comment (sunng87) [inline docs/en/v0.4/greptimecloud/greptimeai/langchain.md:20]: Add code example for how to use `greptime_callback`

File: docs/en/v0.4/greptimecloud/greptimeai/langchain.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+callbacks = [GreptimeCallbackHandler()]
+
+llm = OpenAI()
+prompt = PromptTemplate.from_template("1 + {number} = ")
+chain = LLMChain(llm=llm, prompt=prompt)
+chain.run(number=2, callbacks=callbacks)
+```
+
+You can find more details in [example](https://github.com/GreptimeTeam/greptimeai/blob/main/examples/langchain.ipynb)
```
Comment (xtang) [inline docs/en/v0.4/greptimecloud/greptimeai/langchain.md:35]: @yuanbohan Is the repository named GreptimeAI still private? Should we make it public before merging this pull request?

File: docs/en/v0.4/greptimecloud/greptimeai/langchain.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+and [cookbook](https://github.com/GreptimeTeam/greptimeai-cookbook/tree/main/examples/langchain)
```
Comment (xtang) [inline docs/en/v0.4/greptimecloud/greptimeai/langchain.md:36]: The same applies here regarding the private repository `greptimeai-cookbook`.

**Pull request merged**